### DISCUSSION TITLE
Does BelongsToManyRelationManager not support translation?

### DISCUSSION THREAD

**Discussion by nabileisa** [filamentphp/filament#1895] [Help]
Hello , does the BelongsToManyRelationManager classes not support translation using `spatie-laravel-translatable-plugin`?
because I have been experimenting with filament and I noticed that form fields would save and load translated data correctly in the Resource but when I attempt to do the same using the modal in the relationship manager with the same form it would show  `[object Object]` in the translatable fields,
So , is it not supported or just user error on my part?

**Comment by danharrin** [comment]
https://github.com/laravel-filament/filament/discussions/1514#discussioncomment-2156698
